frontend/src/app/authentication/service/token.service.ts

```diff
@@ -14,7 +14,7 @@ export class TokenService {
         if (token === null) {
             return '';
         } else {
-            return token;
+            return 'Bearer ' + token;
```

Comment:
The Backend already adds the prefix to the token. We can change that in the backend, but I don't think this will work at the moment (I have not tested this).